pkg/commonutils/common_utils.go

```diff
@@ -18,3 +19,13 @@ func MeasureOperation(operation string, log logrus.FieldLogger, metricsApi metri
 		}
 	}
 }
+
+func SaveDiskPartitionsIsSet(installerArgs string) bool {
```

Comment:
Why is this redefined in internal/host/hostcommands/cmd_common.go?

Reply:
actually by mistake, leftover